docs/content/en/integration/openid-connect/roundcube/index.md

```diff
@@ -0,0 +1,231 @@
+//   set the corresponding SSL/TLS options via 'imap_host' and 'imap_conn_options';
+// - For an SMTP instance on localhost, no auth would be required. However
+//   RC's OAuth enforces 'smtp_auth_type' = 'XOAUTH2' plus credentials (cf.
+//   <https://github.com/roundcube/roundcubemail/issues/9183>), thus you *must*
+//   use TLS or SSL via 'smtp_host' and 'smtp_conn_options'!
+```
+
+### Authelia
+
+The following YAML configuration is an example __Authelia__
+[client configuration](../../../configuration/identity-providers/openid-connect/clients.md) for use with [Roundcube]
+which will operate with the above example:
+
+```yaml
+identity_providers:
+  oidc:
+    ## The other portions of the mandatory OpenID Connect 1.0 configuration go here.
+    ## See: https://www.authelia.com/c/oidc
+    clients:
+    - id: 'roundcube'
+      description: 'Roundcube'
+      secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng'  # The digest of 'insecure_secret'.
+      public: false
+      authorization_policy: 'two_factor'
+      redirect_uris:
+        - 'https://roundcube.example.com/oauth/callback/'
+      scopes:
+        - 'openid'
+        - 'profile'
+        - 'email'
+# optional
+access_control:
+  rules:
+    - domain: 'webmail.example.com'
+      resources:
+        - '^/(skins|plugins|jquery|program)([/?].*)?$'
+      policy: bypass
+```
+
+Also, mind that your reverse proxy might need special arrangements -- see an example for Nginx here below.
+
+### Nginx
+
+The [standard authrequest-based
+example](https://www.authelia.com/integration/proxies/nginx/#standard-example)
+applies. However, if you want to use the *same* reverse-proxy instance for
+your protected application, its corresponding vhost might need the following
+adaptations:
+
+```nginx
+# Backend proxied app on the same host
+server {
+    # See N.B. below
+    listen 9092;
+    server_name roundcube;
+
+    ...
+
+    location ~ [^/]\.php(?:/|$) {
+        fastcgi_split_path_info ^(.+?\.php)(/.*)$;
+        if (!-f $document_root$fastcgi_script_name) {
+            return 404;
+        }
+
+        include fastcgi_params;
+
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param PATH_INFO       $fastcgi_path_info;
+
+        # See N.B. below
+        fastcgi_param  SERVER_PORT '';
+
+        fastcgi_pass $your_php_upstream;
+    }
+}
+```
+
+**N.B.** Roundcube's redirect URI is _not_ configurable, but is
+dynamically built with bits coming from the FCGI environment:
+`<scheme>://<fqdn>[:<port>]/...`. To force HTTPS, Roundcube's conf flag
+`use_https` must be set. However, the redirection breaks when the backend
+application is listening on a specific port, because the resulting redirect
+URI would be something like `https://<fqdn>:<port>/...`. Thus, to obtain the
+correct redirect URI `'https://<fqdn>/...'`, the fastcgi_param SERVER_PORT
+must be unset.
```

Comment:
This looks like a good note. It should be located as part of the application section however and should be stylized similar to our other notes. i.e. it should have  a bold and italic `Important Note:` followed by the italic content. Similar to below:

_**Important Note:** Roundcube's Redirect URI is **not** configurable, but is
dynamically built with bits coming from the FCGI environment:
`<scheme>://<fqdn>[:<port>]/...`. To force HTTPS, Roundcube's conf flag
`use_https` must be set. However, the redirection breaks when the backend
application is listening on a specific port, because the resulting redirect
URI would be something like `https://<fqdn>:<port>/...`. Thus, to obtain the
correct redirect URI `'https://<fqdn>/...'`, the fastcgi_param SERVER_PORT
must be unset._

Reply:
OK!